### PR TITLE
Update sumitishere.json

### PR DESCRIPTION
**Pull request by sumitishere**
Updated the social links, please change my urls

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7701"></a>

### CODE REVIEW

File: data/sumitishere.json [hunk 1]
```diff
@@ -21,28 +21,28 @@
     },
     {
       "icon": "FaLinkedin",
```
Comment (CBID2) [inline data/sumitishere.json:23]: [Proofreading] `icon` strings are not supposed to end in a comma.
```suggestion
      "icon": "FaLinkedin"
```

File: data/sumitishere.json [hunk 2]
```diff
@@ -21,28 +21,28 @@
...
+      "url": "https://twitter.com/sumitverse__",
       "icon": "FaTwitter",
       "color": "#00ACEE"
     },
     {
       "group": "Primary Socials",
       "name": "LinkedIn: Connect with me",
-      "url": "https://www.linkedin.com/in/sumit-mukharjee-2b3769227/",
+      "url": "https://www.linkedin.com/in/sumit-mukharjee-623287280",
       "icon": "FaLinkedin",
```
Comment (CBID2) [inline data/sumitishere.json:39]: [Proofreading] See [my feedback on this line](https://github.com/EddieHubCommunity/LinkFree/pull/7701/files#r1235935089) to see how to improve this one. 
```suggestion
      "icon": "FaLinkedin"
```

File: data/sumitishere.json [hunk 3]
```diff
@@ -21,28 +21,28 @@
...
+      "url": "https://www.linkedin.com/in/sumit-mukharjee-623287280"
     }
   ],
   "links": [
     {
       "group": "Primary Socials",
       "name": "Twitter: Follow me",
-      "url": "https://twitter.com/sumitisheree",
+      "url": "https://twitter.com/sumitverse__",
       "icon": "FaTwitter",
```
Comment (CBID2) [inline data/sumitishere.json:32]: [Proofreading] See [my feedback on this line](https://github.com/EddieHubCommunity/LinkFree/pull/7701/files#r1235935089) to see how to improve this one. 
```suggestion
      "icon": "FaTwitter",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @sumitishere! :) I just gave some suggestions on how to improve your profile. I highly recommend referring to the [quickstart](https://linkfree.io/docs/quickstart) when updating it in the future. Other than that, you're off to a great start! 😄  👍🏾

File: data/sumitishere.json (hunk 1 above)
Comment (SaraJaoude) [reply data/sumitishere.json:23]: @CBID2 this is incorrect. The property name does not affect if it should end in a comma so I will mark this as resolved.

File: data/sumitishere.json (hunk 3 above)
Comment (SaraJaoude) [reply data/sumitishere.json:32]: @CBID2 this is incorrect. The property name does not affect if it should end in a comma so I will mark this as resolved.

File: data/sumitishere.json (hunk 2 above)
Comment (SaraJaoude) [reply data/sumitishere.json:39]: @CBID2 this is incorrect. The property name does not affect if it should end in a comma so I will mark this as resolved.

**Comment by SaraJaoude** [comment]
> Hi @sumitishere! :) I just gave some suggestions on how to improve your profile. I highly recommend referring to the [quickstart](https://linkfree.io/docs/quickstart) when updating it in the future. Other than that, you're off to a great start! 😄 👍🏾

@CBID2 please refer to my comments on your inline suggestions - these actually rendered the JSON invalid. Also, if you are suggesting documentation to contributors so they can update their profile, it is best to recommend the [Editing Guide](https://linkfree.io/docs/how-to-guides/editing) (rather than the QuickStart guide - given this is designed to help contributors when they are starting their profile, not updating it in future)

**Review by SaraJaoude — APPROVED** [review]
Looks good

**Comment by CBID2** [comment]
Noted

**Pull request merged**